app/controllers/newflow/login_signup_controller.rb

```diff
@@ -329,7 +329,7 @@ def change_password
     def exit_accounts
       referrer_params = extract_params(request.referrer)
 
-      if (r = referrer_params[:r])
+      if (r = referrer_params[:r]) # `r`edirect parameter. See `before_action :save_redirect`.
```

Comment:
why cant this just be `if (redirect_param = referrer_params[:r])` and not need the comment?

Reply:
Good question; sure, I can do that.

Reply:
Done.